store/hbase/kv.go

```diff
@@ -25,14 +24,14 @@ import (
 )
 
 const (
-	// ColFamily is the hbase column family name.
-	ColFamily = "f"
-	// Qualifier is the hbase column name.
-	Qualifier = "q"
-	// FmlAndQual is a shortcut.
-	FmlAndQual = ColFamily + ":" + Qualifier
-	// HbaseTableName is the table name in hbase which is invisible to SQL layer.
-	HbaseTableName = "tidb"
+	// hbaseColFamily is the hbase column family name.
+	hbaseColFamily = "f"
```

Comment:
You may create both  hbaseColFamilyStr and hbaseColFamilyBytes, so you don't have to convert it in Get method.